docs/summer-of-code-ideas/2019-JiradetOunjai.md

```diff
@@ -0,0 +1,140 @@
+# Google Summer of Code 2019 Report - Jiradet Ounjai
+
+The Google Summer of Code program has turned out to be a great learning opportunity for me. I gain fresh knowledge from my fantastic mentors through the discussions and the intense code reviews we had. I would like to thank my mentors for the time and energy you put into helping me.
+
+I have made this document to summarize the work done for my Google Summer of Code project *Improve Shading Language Support in GraphicsFuzz*. 
+
+
+## Personal Links
+[Github](https://github.com/jiradeto) - [Linkedin](https://www.linkedin.com/in/jiradeto/)
+
+
+## Project Link
+[Improve Shading Language Support in GraphicsFuzz](https://summerofcode.withgoogle.com/projects/#4859963594244096)
+
+
+## Mentors
+Alastair Donaldson, Paul Thomson
+
+
+## Deliverables
```

Comment:
"providing a new shading language support" sounds like you were supporting a new language altogether; rephrase to talk about making shading language support more complete.